src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const i in robot) {
+    if (result[robot[i]]) {
```

Comment:
Value can be 0. So you rewrite this value but should return null. Use hasOwnProperty.